package/toltec-bootstrap/toltecctl

```diff
@@ -169,48 +313,72 @@ uninstall-force() {
     reboot
```

Comment:
```suggestion
    read -p "To complete the uninstall process, the device needs to be rebooted. Reboot now? [Y/n]" -n 1 -r
    echo

    if [[ ! $REPLY =~ ^[Nn]$ ]]; then
        echo "Please reboot your device manually to complete the uninstall process."
        return
     fi
    reboot
```

Tidbit: Not sure if user could get confused when the uninstall just reboots the device. Maybe add a prompt to ask the user to confirm it?

Reply:
Perhaps also prompt to print the SSH password before reboot just in case they don't have it written down?